cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -74,7 +73,8 @@ def _provisioner(template, provisioning_data, delayed=None):
         vm = Vm(name=vm_name, provider=provider, template_name=template)
         view = navigate_to(vm, 'Provision')
         view.form.fill_with(provisioning_data, on_change=view.form.submit_button)
-        flash.assert_no_errors()
+        base_view = vm.appliance.browser.create_view(BaseLoggedInPage)
```

Comment:
**Question**: Maybe better use RequestsView here, as user after submitting redirects to Requests page

Reply:
Not if there is an error